h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -578,4 +583,20 @@ public void done(Frame frame) {
     }
   }
 
+  static Consumer<Vec> dropit = new Consumer<Vec>() {
+    @Override public void accept(Vec v) { v.remove(new Futures()).blockForPending(); }
+  };
+
+  @BeforeClass
+  public static void hi() { stall_till_cloudsize(1); }
+  @AfterClass public static void bye() { toDrop.forEach(dropit); }
+
+  private static Set<Vec> toDrop = new HashSet<>();
+
+  protected static Vec willDrop(Vec v) { toDrop.add(v); return v; }
+
+  protected static <T extends Vec.VectorHolder> T willDrop(T vh) {
```

Comment:
VectorHolder interface is only used in tests. In this case I don't think it make sense to introduce a new interface and make the code base more complex. You can add function willDrop(Chunk chk) and accomplish the same.